exercises/kindergarten-garden/canonical-data.json

```diff
@@ -0,0 +1,127 @@
+{
+  "exercise": "kindergarten-garden",
+  "version": "1.0.0",
+  "comments": [
+    "Garden can be created with just a diagram or a diagram and students.",
+    "Students' gardens are assigned in alphabetical order.",
+    "The default students are:",
+    "  Alice, Bob, Charlie, David, Eve, Fred, Ginny, Harriet, Ileana, Joseph, Kincaid, and Larry."
```

Comment:
since this information is also in the https://github.com/exercism/x-common/blob/master/exercises/kindergarten-garden/description.md, any chance of it falling out date? I try to reduce duplication when possible. I am thinking it doesn't need to be put here.